.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,5 +1,22 @@
-Fixes #[Add issue number here. Note: This will automatically close the issue. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+<!-- 
+(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
+-->
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+#### Checklist
 
-Screenshots for the change: 
+- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
+- [ ] My branch is up-to-date with the Upstream `development` branch.
+- [ ] The acceptance, integration, unit tests and linter pass locally with my changes
```

Comment:
remove this line

Reply:
Which line number?

Reply:
@saarthakchats Line number 9. It is mentioned on the left side.